index.js

```diff
@@ -31,12 +34,23 @@ function UpdateNotifier(options) {
 	this.callback = options.callback || function () {};
 
 	if (!this.hasCallback) {
-		this.config = new Configstore('update-notifier-' + this.packageName, {
-			optOut: false,
-			// init with the current time so the first check is only
-			// after the set interval, so not to bother users right away
-			lastUpdateCheck: Date.now()
-		});
+		try {
+			this.config = new Configstore('update-notifier-' + this.packageName, {
+				optOut: false,
+				// init with the current time so the first check is only
+				// after the set interval, so not to bother users right away
+				lastUpdateCheck: Date.now()
+			});
+		} catch (_) {
```

Comment:
@SBoudrias I'm not quite sure what you mean; I used `_` as the var name for the error to signify that it's ignored. Could you please clarify?